UnitySDK/Assets/ML-Agents/Scripts/BrainParameters.cs

```diff
@@ -5,8 +5,8 @@ namespace MLAgents
 {
     public enum SpaceType
     {
-        discrete,
-        continuous
+        Discrete,
```

Comment:
(as discussed on slack) Mention this in the upgrade guide; it's probably the only public-facing change that people are likely to hit.

Reply:
where is said upgrade guide?

Reply:
done